src/components/SimpleTable.jsx

```diff
@@ -104,7 +104,7 @@ const DataRows = ({rowData, baseStyle, columnHeaders, rowWrapper = ({renderedRow
           let output;
           //columnHeaders determine width of the columns,
           //therefore extract width from columnHeader and apply to cell style
-          const columnWidthStyle = columnHeaders[cellIndex].cellStyle;
+          const columnWidthStyle = { width: columnHeaders[cellIndex].cellStyle.width };
```

Comment:
I didn't know where to check where this was used - but I was noticing we said we were extracting just the width, but actually pulling everything. I figured this would be cleaner.